debian/control

```diff
@@ -0,0 +1,25 @@
+Source: ddfacet
+Section: science
+Priority: extra
+Maintainer: Debian Astro Team <debian-astro-maintainers@lists.alioth.debian.org>
+Build-Depends: cmake,
+               gfortran,
+               debhelper (>=9),
+               python-setuptools,
```

Comment:
you need to specify the python dependencies as build dependencies otherwise the dependency finding logic can't find the packages :)

Reply:
Done!